README.md

```diff
@@ -0,0 +1,11 @@
+# Pharo In Progress
+
+These chapters are part of [an experiment](https://github.com/SquareBracketAssociates/PharoForTheEnterprise-english) of writing a book with the Pier syntax using plain-text editors. Once they are ready for review we can move them to Pharo-ReadyForReviews.
+
+This book is continuously built on an [Inria Jenkins server](https://ci.inria.fr/pharo-contribution/job/PharoBookWorkInProgress/).
+
+
+How to write
+------------
+
+This book is written in Pillar markup. If you are not familiar with it please check the [pillar-documentation](https://github.com/pillar-markup/pillar-documentation).
```

Comment:
please reference Pillar website on smalltalkhub instead: http://www.smalltalkhub.com/#!/~Pier/Pillar. This page always contains un up-to-date link to documentation.